pypeit/core/flux_calib.py

```diff
@@ -56,27 +54,23 @@ def find_standard_file(ra, dec, toler=20.*units.arcmin, check=False):
             star exists within the input ra, dec, and toler range.
 
     Returns:
-        dict, bool: If check is True, return True or False depending on
-        if the object is matched to a library standard star.  If check
-        is False and no match is found, return None.  Otherwise, return
+        dict or bool: If check is True, return True or False depending on
+        if the object is matched to a library standard star.
+        If check is False and no match is found, return None.  Otherwise, return
         a dictionary with the matching standard star with the following
         meta data:
 
-            - 'file': str -- Filename table
+            - 'cal_file': str -- Filename table
             - 'name': str -- Star name
-            - 'ra': str -- RA(J2000)
-            - 'dec': str -- DEC(J2000)
+            - 'std_ra': str -- RA(J2000)
+            - 'std_dec': str -- DEC(J2000)
 
     """
     # Priority
     std_sets = ['xshooter', 'calspec', 'esofil']
 
     # SkyCoord
-    try:
-        ra, dec = float(ra), float(dec)
-        obj_coord = coordinates.SkyCoord(ra, dec, unit=(units.deg, units.deg))
-    except:
-        obj_coord = coordinates.SkyCoord(ra, dec, unit=(units.hourangle, units.deg))
+    obj_coord = coordinates.SkyCoord(ra, dec, unit='deg')
```

Comment:
Is there a reason why we are removing support for multiple formats? Did something change upstream?